packages/ipfs-core/src/components/files/utils/remove-link.js

```diff
@@ -127,7 +127,7 @@ const updateShard = async (context, positions, child, options) => {
 
   log(`Updating shard ${prefix} with name ${newName}`)
 
-  const size = DAGNode.isDAGNode(result.node) ? result.node.size : result.node.Tsize
+  const size = result.node instanceof DAGNode ? result.node.size : result.node.Tsize
```

Comment:
I always get told off by browser people for using `instanceof`, isn't this something we're supposed to avoid by duck typing at a minimum?